main/src/main/scala/sbt/Keys.scala

```diff
@@ -221,7 +221,7 @@ object Keys {
   val allowMachinePath = settingKey[Boolean]("Allow machine-specific paths during conversion.")
   val rootPaths = settingKey[Seq[NioPath]]("The root paths used to abstract machine-specific paths.")
```

Comment:
Would it make sense to move this to `sbt.nio.Keys`? Also, is `Seq[NioPath]` the right abstraction here? It seems a bit strange that these are order dependent. I assume that the index into this `Seq` corresponds to the `${N}/path/to/file` in VirtualFileRefs? It seems like it might be a bit more robust to have virtual files that look like `${base}/path/to/file` instead at the cost of a modest increase in storage.

Reply:
I split the VirtualFileRef suggestion into an issue - https://github.com/sbt/zinc/issues/772

As per sbt.Keys, if the key is meant to be used by build authors then I think we should mostly keep them in Keys so they can `import sbt.Keys._`.